### ISSUE TITLE
`npx strapi develop --watch-admin` doesn't work in Firefox (NS_BINDING_ABORTED)

### ISSUE THREAD

**Issue by erikschul** [strapi/strapi#17083]
### Required System information

- Node.js version: v18
- Strapi version: v4.11.2
- Operating system (Docker): Linux
- Is your project Javascript or Typescript: TypeScript

Firefox v114.0.2
Operation system (browser): MacOS Ventura

### Describe the bug

`npx strapi develop --watch-admin` doesn't work in Firefox, but works in Chrome.

Inspector tab shows the network request as failing with "NS_BINDING_ABORTED".

<img width="954" alt="image" src="https://github.com/strapi/strapi/assets/128804597/80f2312f-6007-4bc7-ac60-921a5d162977">

This may be an issue/incompatibility between Webpack/Firefox.

### Steps to reproduce the behavior

```
npx create-strapi-app@latest bughunt
> Quickstart
> Create user via browser
npx strapi develop --watch-admin
```

**Comment by joshuaellis** [comment]
I would check no plugins are interfering.

**Comment by erikschul** [comment]
I restarted, and it works now. Maybe the problem was with Docker.
But, the issue persists with some requests. Although I think I saw the same when I was working on Next.js, with random requests failing. After opening the specific resource in a new tab, and refreshing, it no longer fails with NS_BINDING_ERROR.

I will close the ticket. Thanks for the help!

<img width="886" alt="image" src="https://github.com/strapi/strapi/assets/128804597/8230b9a2-9847-43a4-9e01-0d06a2d711ea">
